tests/lir_tests/helpers.rs

```diff
@@ -0,0 +1,121 @@
+// SPDX-License-Identifier: Apache-2.0
+
+use indexmap::IndexMap;
+use num_bigint::BigInt;
+use solang::{
+    lir::{
+        expressions::{BinaryOperator, Expression, Operand, UnaryOperator},
+        lir_type::Type,
+        printer::Printer,
+        vartable::Vartable,
+    },
+    sema::ast,
+};
+use solang_parser::pt::Loc;
+
+pub(crate) fn binop_expr(left: Operand, op: BinaryOperator, right: Operand) -> Expression {
+    Expression::BinaryExpr {
+        loc: Loc::Codegen,
+        operator: op,
+        left: Box::new(left),
+        right: Box::new(right),
+    }
+}
+
+pub(crate) fn unop_expr(op: UnaryOperator, right: Operand) -> Expression {
+    Expression::UnaryExpr {
+        loc: Loc::Codegen,
+        operator: op,
+        right: Box::new(right),
+    }
+}
+
+pub(crate) fn num_literal(value: i32, signed: bool, width: u16) -> Operand {
+    Operand::NumberLiteral {
+        value: BigInt::from(value),
+        ty: if signed {
+            Type::Int(width)
+        } else {
+            Type::Uint(width)
+        },
+        loc: Loc::Codegen,
+    }
+}
+
+#[macro_export]
+macro_rules! stringfy_expr {
+    ($printer:expr, $expr:expr) => {{
+        let mut buffer = Vec::new();
+        $printer.print_expr(&mut buffer, $expr);
+        String::from_utf8(buffer).expect("Failed to convert to string")
+    }};
+}
+
+#[macro_export]
+macro_rules! stringfy_insn {
+    ($printer:expr, $insn:expr) => {{
+        let mut buf = Vec::new();
+        $printer.print_insn(&mut buf, $insn);
+        String::from_utf8(buf).unwrap()
```

Comment:
Can the buffer be the string already?

Reply:
Note: will keep this unchanged, as it is only meant for testing purposes.